cores/esp8266/core_esp8266_phy.cpp

```diff
@@ -288,20 +289,10 @@ static const uint8_t ICACHE_FLASH_ATTR phy_init_data[128] =
     /*[114] =*/ 1
 };
 
-
-// These functions will be overriden from C++ code.
-// Unfortunately, we can't use extern "C" because Arduino preprocessor
-// doesn't generate forward declarations for extern "C" functions correctly,
-// so we use mangled names here.
-#define __get_adc_mode _Z14__get_adc_modev
-#define __get_rf_mode _Z13__get_rf_modev
-#define __run_user_rf_pre_init _Z22__run_user_rf_pre_initv
-
```

Comment:
Does removal mean the comment is addressed for the sketch .ino having  `void __get_adc_mode() { return myvalue; }` func that is without the `extern "C"`?

Reply:
These functions are now properly prototyped in `coredecls.h`, as such, yes, external C linkage.

Reply:
non-void `int __get_adc_mode();` - smile ;-)